loader/common/log.c

```diff
@@ -13,6 +13,64 @@ enum log_level logger_set_level(enum log_level level)
     return prev;
 }
 
+#ifdef HYPER_SERIAL_LOG
+static inline void out8(u16 port, u8 data)
```

Comment:
Technically we already have an `io.h` header in `tests/kernel/include/common/io.h`

Can we move it to loader `include/common/` in a separate commit since it's now needed here?
Test kernel CMakeLists.txt already fetches a bunch of loader headers, so should be relatively easy.